### ISSUE TITLE
Setup Google Analytics for devfest.gdgpisa.it

### ISSUE THREAD

**Issue by cortinico** [gdgpisa/hoverboard#1]
We need to setup a proper Google Analytics account and update the [Json file](https://github.com/gdgpisa/hoverboard/blob/master/data/hoverboard.config.json#L6).

```
  "analytics": {
    "prod": "UA-43643469-8",
    "dev": "UA-43643469-9"
  },
```

**Comment by domenicoblanco** [comment]
I'll do it in a few minutes!

**Comment by cortinico** [comment]
Closed with #4 <3
